source/pcb/status_display_pcb.rst

```diff
@@ -0,0 +1,125 @@
+==========================
+VantTec Status Display PCB
+==========================
+
+Description
+===========
+
+This is a 2-layer PCB designed as a complementary device for our team in order to get information more easily about the status of the vehicle, such as sensors or battery power percentage. 
```

Comment:
igual puede ser:
as a complementary device for our team to displau information about the status ...

además, puedes añadir el link del repo?